tests/unit/tuf_repository_service_worker/test_repository.py

```diff
@@ -438,11 +454,11 @@ def test__update_snapshot_specific_targets(self, test_repo, monkeypatch):
             pretend.call(mocked_bins_md, "bins-e"),
         ]
 
-    def test_update_snapshot_bump_all(self, test_repo, monkeypatch):
+    def test__update_snapshot_bump_all(self, test_repo, monkeypatch):
         snapshot_version = 3
         mocked_snapshot = pretend.stub(
             signed=pretend.stub(
-                meta={},
+                meta={"bins-e.json": 3, "bins-f.json": 3},
```

Comment:
Should be version 4 as on line 466 you define it as such:
```suggestion
                meta={"bins-e.json": 4, "bins-f.json": 4},
```

Reply:
This is the initial value in the current snapshot and for that moment it has version 3 for the bin roles in the meta.

On line 466, you're getting the bins Metadata which was already bumped and persisted.
https://github.com/repository-service-tuf/repository-service-tuf-worker/blob/57220c9973676d2ff1fbc96929df35b26c448c1d/repository_service_tuf_worker/repository.py#L383
That is why I changed the mock of `_bump_and_persist` to simulate it.
And you want to see the new version in the meta.